packages/fuel-indexer/src/executor.rs

```diff
@@ -657,17 +675,44 @@ impl WasmIndexExecutor {
             }
         }
     }
+
+    pub async fn get_instance_metering_points(
```

Comment:
@lostman Going forward, we're trying to add `rustdoc` comments to any new functions we create or any previous functions/methods that we change. Can we do that here?

Makes the code much more easy to follow for others.